checks/raw/testdata/Dockerfile-curl-sh

```diff
@@ -19,6 +19,8 @@ RUN echo hello && curl -s /etc/file | sh
 RUN echo hello && curl -s file-with-sudo2 | sudo bash
 RUN echo hello && sudo curl -s file-with-sudo | bash | bla
 RUN ["echo", "hello", "&&", "curl", "-s", "/etc/file2", "|", "sh"]
+RUN curl -sSL https://raw.githubusercontent.com/dotnet/install-scripts/main/src/dotnet-install.sh | bash /dev/stdin
+RUN curl -sSL https://raw.githubusercontent.com/dotnet/install-scripts/5b142a1e445a6f060d6430b661408989e9580b85/src/dotnet-install.sh | bash /dev/stdin
```

Comment:
can you add the `#pinned` and `#unpinned` comments here too? found it helpful in the other file